oauth2_http/java/com/google/auth/oauth2/ComputeEngineCredentials.java

```diff
@@ -334,13 +338,20 @@ public IdToken idTokenWithAudience(String targetAudience, List<IdTokenProvider.O
     return IdToken.create(rawToken);
   }
 
-  private HttpResponse getMetadataResponse(String url) throws IOException {
+  private HttpResponse getMetadataResponse(String url, RequestType requestType) throws IOException {
     GenericUrl genericUrl = new GenericUrl(url);
     HttpRequest request =
         transportFactory.create().createRequestFactory().buildGetRequest(genericUrl);
     JsonObjectParser parser = new JsonObjectParser(OAuth2Utils.JSON_FACTORY);
     request.setParser(parser);
     request.getHeaders().set(METADATA_FLAVOR, GOOGLE);
+    // do not send metric header for getUniverseDomain and getAccount
+    if (requestType != RequestType.UNTRACKED) {
```

Comment:
Just to confirm: this is not sending *any* header. So we also won't be sending `MetricsUtils.getLanguageAndAuthLibraryVersions()`. This is intended right?

Reply:
Ok gotcha, I think that makes sense. Thanks!